src/renderer/components/+nodes/nodes.tsx

```diff
@@ -56,6 +56,13 @@ enum columnId {
 interface Props extends RouteComponentProps<NodesRouteParams> {
 }
 
+interface UsageArgs {
+  node: Node;
+  title: string;
+  metricNames: [string, string];
+  secondFormater: (metrics: [number, number]) => string;
```

Comment:
Should this be spelled `secondFormatter`? 

Maybe the argument name would be a bit clearer if both formatter would be passed, even though it's same for all of them for now.

Something like `primaryFormatter` `secondaryFormatter`?

Reply:
> Should this be spelled `secondFormatter`?

yes

> Maybe the argument name would be a bit clearer if both formatter would be passed, even though it's same for all of them for now.

sure.